### ISSUE TITLE
Add more OpenCV examples

### ISSUE THREAD

**Issue by madduci** [blackibiza/opencv_demo#1]
It would be great to add more examples, e.g. one for each algorithm / module, to make the study and understanding of OpenCV framework easier

**Comment by giantpanpan** [comment]
Hi! Can I still work on this issue? I would like to add hsv histgram feature extraction.

**Comment by madduci** [comment]
@giantpanpan yeah sure, every contribution is welcome! Can you try to make it work for OpenCV 2.4 and 3.x?

**Comment by giantpanpan** [comment]
@madduci Sorry for replying you late! Is that ok is I use opencv 3.4.2?

**Comment by madduci** [comment]
Sorry for the late reply @giantpanpan , OpenCV 3.4.2 is fine
